test/e2e/token_inactivity_timeout_test.go

```diff
@@ -103,12 +105,11 @@ func TestTokenInactivityTimeout(t *testing.T) {
 		checkTokenAccess(t, userClient, oauthClientClient, configInactivityTimeout, nil, testTokenTimeouts)
 	})
 
-	updateOAuthConfigInactivityTimeout(t, configClient, &metav1.Duration{Duration: time.Duration(configInactivityTimeout) * time.Second})
-	test.WaitForKubeAPIServerStartProgressing(t, configClient)
-	test.WaitForKubeAPIServerClusterOperatorAvailableNotProgressingNotDegraded(t, configClient)
-
 	// With only OAuth config timeout.
 	t.Run("with-inactivity-timeout", func(t *testing.T) {
+		updateOAuthConfigInactivityTimeout(t, configClient, &metav1.Duration{Duration: time.Duration(configInactivityTimeout) * time.Second})
+		test.WaitForKubeAPIServerStartProgressing(t, configClient)
+		test.WaitForAPIServerToStabilizeOnTheSameRevision(t, kubeClient.CoreV1().Pods("openshift-kube-apiserver"))
```

Comment:
why does it help to just move this into the subtest? The order of execution is the same, isn't it?

Reply:
Yes, the order of execution is the same. But if the method waiting for apiserver to be same revision fails, it would now fail under this subtest. Otherwise the failure is not attributed to this test directly. 

https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_cluster-kube-apiserver-operator/928/pull-ci-openshift-cluster-kube-apiserver-operator-master-e2e-aws-operator/1292778219547660288/artifacts/e2e-aws-operator/junit_report.xml

A minor change to improve the debugging.

Reply:
In the link I posted, helper method failed before this subtest. but a direct look at the report does not show the same.